src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+**ACR**
+
+* Move to 2019-05-01 api-version for ACR resources.
```

Comment:
Though it has been planned since March, but still a breaking change I would recommend to call out. Maybe add an entry like `BREAKING CHANGE: Removed classic registries support`?

Reply:
Done.